scripts/scripts.js

```diff
@@ -481,7 +481,7 @@ window.store = new (class {
           if (!this._queryMap[query]) {
             this._queryMap[query] = {
               spreadsheet,
-              limit: 0,
+              limit: 50,
```

Comment:
Mhh not sure why you need to set the default limit to 50 ?

Reply:
having a default of zero will return an empty list, its quite a confusing extra step to consider when implementing a new query

Reply:
@DriftingSands  For which block does it return an empty list ? If you're creating a new block that is supposed to support queries, you'll have to add it here https://github.com/headwirecom/helix-sportsmagazine/blob/55a7d002d8ea08184e6067572e6bcac7229c6b0e/scripts/scripts.js#L461-L466 with a limit configured to the max data items it can hold.